app/assets/config/manifest.js

```diff
@@ -1,2 +1,6 @@
 //= link_tree ../images
 //= link_directory ../stylesheets .css
+//= link_directory ../stylesheets .scss
+//= link application.scss
+
+import "bootstrap"
```

Comment:
read how manifest.js works and fix it. 
for quick testing, you can checkout from this branch and test fixes on heroku or use` RAILS_ENV=production rails assets:precompile `and see how the assets are built